test/unit/vl-map-search.test.html

```diff
@@ -325,6 +322,207 @@
         assert.equal(element.getTranslation('select.no_more_options'), text4);
       });
     });
+
+    suite('Lambert-coördinaat', () => {
+      test('of - een vrije tekst, geeft "undefined"', () => {
+        expect(LambertCoordinaat.of('test')).to.be.undefined;
+      });
+
+      test('of - "null", geeft "undefined"', () => {
+        expect(LambertCoordinaat.of(null)).to.be.undefined;
+      });
+
+      test('of - "undefined", geeft "undefined"', () => {
+        expect(LambertCoordinaat.of(undefined)).to.be.undefined;
+      });
+
+      test('of - een lege string, geeft "undefined"', () => {
+        expect(LambertCoordinaat.of('')).to.be.undefined;
+      });
+
+      test('of - een geldige coördinaat, geeft een LambertCoordinaat', () => {
+        const lambertCoordinaat = LambertCoordinaat.of('123456.78, 345232.04');
+
+        expect(lambertCoordinaat).not.to.be.undefined;
+        expect(lambertCoordinaat).to.deep.equal({
+          _x: 123456.78,
+          _y: 345232.04,
+        });
+        expect(lambertCoordinaat.toString()).to.be.equal('123456.78, 345232.04');
+      });
+
+      test('of - een geldige coördinaat met gehele getallen, geeft een LambertCoordinaat', () => {
+        const lambertCoordinaat = LambertCoordinaat.of('123456, 34523');
+
+        expect(lambertCoordinaat).not.to.be.undefined;
+        expect(lambertCoordinaat).to.deep.equal({
+          _x: 123456,
+          _y: 34523,
+        });
+        expect(lambertCoordinaat.toString()).to.be.equal('123456, 34523');
+      });
+
+      test('of - geldige coordinaat met haakjes en gescheiden door een puntkomma, geeft een LambertCoordinaat', () => {
+        const lambertCoordinaat = LambertCoordinaat.of('(123456.78; 345232.04)');
+
+        expect(lambertCoordinaat).not.to.be.undefined;
+        expect(lambertCoordinaat).to.deep.equal({
+          _x: 123456.78,
+          _y: 345232.04,
+        });
+        expect(lambertCoordinaat.toString()).to.be.equal('123456.78, 345232.04');
+      });
+    });
+
+    suite('Lambert-coördinaat - RegEx', () => {
+      test('Matched met: een geldige coördinaat', () => {
+        const searchInput = '104719.27, 192387.25';
+        const result = searchInput.match(LAMBERT_COORDINAAT_REGEX);
```

Comment:
Ik denk dat je hier niet de regex wil exposen en daar rechtstreeks tegen wil testen, maar wil testen tegen de factory method (of) van Lambert coordinaat.